.github/workflows/translate.yml

```diff
@@ -0,0 +1,38 @@
+name: Translate Reports
+
+on:
+  workflow_call:
+    inputs:
+      environment:
+        description: The Github environment to load secrets from
+        type: string
+        required: true
+
+jobs:
+  translate:
+    environment: ${{ inputs.environment }}
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: lts/*
+
+      - name: Install NPM dependencies
+        run: npm ci
+        working-directory: site/gatsby-site
+
+      - name: Run translation script
+        run: node src/scripts/translateReports.js
```

Comment:
nit. I prefer using package.json scripts, which makes it easier to add/remove parameters, etc.

Reply:
done!